manifests/rc.pp

```diff
@@ -0,0 +1,24 @@
+# === Copyright
+#
+# Copyright 2014, Deutsche Telekom AG
+# Licensed under the Apache License, Version 2.0 (the "License");
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+
+# == Class: os_hardening::rc
+#
+# Configures /etc/init.d/rc
+#
+class os_hardening::rc (
+  $umask = '027',
+){
+  # set the file
+  file {
+    '/etc/init.d/rc':
```

Comment:
can you maybe elaborate why this is required? what about systemd, is it related/important here?